### PR TITLE
Enable Analytics v2

### PR DESCRIPTION
**Pull request by eessex**
**HOLD** for https://github.com/artsy/reaction/pull/3193, major reaction upgrade

Updates all force auth entrypoints to include all required analytics fields. 

This has been fully QA'd to prove that auth entrypoints are schema compliant, cc @abhitip 

Review app:
https://auth-analytics.artsy.net/

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `dangerfile.ts`.
  </td>
    </tr>
  </tbody>
</table>



## Error GraphQLError
```
Syntax Error: Cannot parse the unexpected character "<".
GraphQLError: Syntax Error: Cannot parse the unexpected character "<".
    at syntaxError (/home/circleci/project/node_modules/graphql/error/syntaxError.js:24:10)
    at readToken (/home/circleci/project/node_modules/graphql/language/lexer.js:302:32)
    at Object.lookahead (/home/circleci/project/node_modules/graphql/language/lexer.js:61:43)
    at Object.advanceLexer [as advance] (/home/circleci/project/node_modules/graphql/language/lexer.js:52:33)
    at expect (/home/circleci/project/node_modules/graphql/language/parser.js:1296:11)
    at parseDocument (/home/circleci/project/node_modules/graphql/language/parser.js:107:3)
    at parse (/home/circleci/project/node_modules/graphql/language/parser.js:38:10)
    at buildSchema (/home/circleci/project/node_modules/graphql/utilities/buildASTSchema.js:447:43)
    at _callee3$ (/home/circleci/project/scripts/validateSchemas.js:154:31)
    at tryCatch (/home/circleci/project/node_modules/regenerator-runtime/runtime.js:45:40)
```
### Dangerfile
```
19|   const changes: string[] = await getBreakingChanges(versionToCheck, 2)
20| 
21|   // There are breaking changes with the schema
22|   if (changes.length) {
23|     if (isReleasePR) {
------------^
24|       const deployURL = "https://github.com/artsy/metaphysics#deployment"
25|       fail(
26|         `You need to promote Metaphysics from staging to production. The schema used in Reaction, is further ahead than the version in Metaphyisc production. This could cause your queries to fail. You can see how to [deploy here](${deployURL}).`
27|       )
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e59755ebfd46649d9a7ff36534c63bb26fbfdd9f
</p>

File: package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "@artsy/gemup": "0.0.3",
     "@artsy/palette": "7.6.0",
     "@artsy/passport": "1.1.11",
-    "@artsy/reaction": "25.42.5",
+    "@artsy/reaction": "25.42.0-canary.3303.56199.0",
```
Comment (damassi) [inline package.json:79]: Yay @zephraph

File: src/desktop/apps/gene/client.js [hunk 2]
```diff
@@ -93,5 +100,3 @@ function setupGenePage() {
   // Setup blurb
   blurb($(".js-gene-blurb"), { limit: 250 })
 }
-
-export default { setupGenePage }
```
Comment (damassi) [inline src/desktop/apps/gene/client.js:97]: Thanks for improving this stuff as you go along 👍

File: src/desktop/apps/partner2/client/view.coffee [hunk 3]
```diff
@@ -15,6 +15,7 @@ ContactView = require './contact.coffee'
 tablistTemplate = -> require('../templates/tablist.jade') arguments...
 mediator = require '../../../lib/mediator.coffee'
 { Following, FollowButton } = require '../../../components/follow_button/index.coffee'
+{ ContextModule } = require "@artsy/reaction/dist/Artsy/Analytics/v2/Schema"
```
Comment (damassi) [inline src/desktop/apps/partner2/client/view.coffee:18]: Looking over this PR and seeing how many use-cases we have for importing directly from a sub namespace, it makes sense. So revising my other comment, only concerned about avoiding `* as` (not actionable now of course)

**Pull request merged**